### ISSUE TITLE
Not building with Qt 5.10

### ISSUE THREAD

**Issue by sithlord48** [myst6re/hyne#11]
Failure Below. (Full log: https://travis-ci.org/myst6re/hyne/jobs/391750367)
```
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DHYNE_TASKBAR_FAKE -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I../../hyne -I. -I/opt/qt510/include -I/opt/qt510/include/QtWidgets -I/opt/qt510/include/QtGui -I/opt/qt510/include/QtCore -I. -isystem /usr/include/libdrm -I/opt/qt510/mkspecs/linux-g++ -o Metadata.o ../Metadata.cpp
../Metadata.cpp: In member function ‘bool Metadata::open()’:
../Metadata.cpp:51:74: error: call of overloaded ‘compare(const char [9], Qt::CaseSensitivity)’ is ambiguous
    if(!isInSaveFile && xml.name().compare("savefile", Qt::CaseInsensitive) == 0) {
                                                                          ^
../Metadata.cpp:51:74: note: candidates are:
In file included from /opt/qt510/include/QtCore/qobject.h:47:0,
                 from /opt/qt510/include/QtCore/qabstractanimation.h:43,
                 from /opt/qt510/include/QtCore/QtCore:5,
                 from ../Metadata.h:21,
                 from ../Metadata.cpp:18:
/opt/qt510/include/QtCore/qstring.h:1632:12: note: int QStringRef::compare(const QString&, Qt::CaseSensitivity) const
 inline int QStringRef::compare(const QString &s, Qt::CaseSensitivity cs) const Q_DECL_NOTHROW
            ^
/opt/qt510/include/QtCore/qstring.h:1563:9: note: int QStringRef::compare(const QByteArray&, Qt::CaseSensitivity) const
     int compare(const QByteArray &s, Qt::CaseSensitivity cs = Qt::CaseSensitive) const
         ^
../Metadata.cpp:69:59: error: call of overloaded ‘compare(const char [10], Qt::CaseSensitivity)’ is ambiguous
     if(xml.name().compare("timestamp", Qt::CaseInsensitive) == 0) {
                                                           ^
../Metadata.cpp:69:59: note: candidates are:
In file included from /opt/qt510/include/QtCore/qobject.h:47:0,
                 from /opt/qt510/include/QtCore/qabstractanimation.h:43,
                 from /opt/qt510/include/QtCore/QtCore:5,
                 from ../Metadata.h:21,
                 from ../Metadata.cpp:18:
/opt/qt510/include/QtCore/qstring.h:1632:12: note: int QStringRef::compare(const QString&, Qt::CaseSensitivity) const
 inline int QStringRef::compare(const QString &s, Qt::CaseSensitivity cs) const Q_DECL_NOTHROW
            ^
/opt/qt510/include/QtCore/qstring.h:1563:9: note: int QStringRef::compare(const QByteArray&, Qt::CaseSensitivity) const
     int compare(const QByteArray &s, Qt::CaseSensitivity cs = Qt::CaseSensitive) const
         ^
../Metadata.cpp:79:66: error: call of overloaded ‘compare(const char [10], Qt::CaseSensitivity)’ is ambiguous
     } else if(xml.name().compare("signature", Qt::CaseInsensitive) == 0) {
                                                                  ^
../Metadata.cpp:79:66: note: candidates are:
In file included from /opt/qt510/include/QtCore/qobject.h:47:0,
                 from /opt/qt510/include/QtCore/qabstractanimation.h:43,
                 from /opt/qt510/include/QtCore/QtCore:5,
                 from ../Metadata.h:21,
                 from ../Metadata.cpp:18:
/opt/qt510/include/QtCore/qstring.h:1632:12: note: int QStringRef::compare(const QString&, Qt::CaseSensitivity) const
 inline int QStringRef::compare(const QString &s, Qt::CaseSensitivity cs) const Q_DECL_NOTHROW
            ^
/opt/qt510/include/QtCore/qstring.h:1563:9: note: int QStringRef::compare(const QByteArray&, Qt::CaseSensitivity) const
     int compare(const QByteArray &s, Qt::CaseSensitivity cs = Qt::CaseSensitive) const
         ^
../Metadata.cpp:87:73: error: call of overloaded ‘compare(const char [9], Qt::CaseSensitivity)’ is ambiguous
    if(isInSaveFile && xml.name().compare("savefile", Qt::CaseInsensitive) == 0) {
                                                                         ^
../Metadata.cpp:87:73: note: candidates are:
In file included from /opt/qt510/include/QtCore/qobject.h:47:0,
                 from /opt/qt510/include/QtCore/qabstractanimation.h:43,
                 from /opt/qt510/include/QtCore/QtCore:5,
                 from ../Metadata.h:21,
                 from ../Metadata.cpp:18:
/opt/qt510/include/QtCore/qstring.h:1632:12: note: int QStringRef::compare(const QString&, Qt::CaseSensitivity) const
 inline int QStringRef::compare(const QString &s, Qt::CaseSensitivity cs) const Q_DECL_NOTHROW
            ^
/opt/qt510/include/QtCore/qstring.h:1563:9: note: int QStringRef::compare(const QByteArray&, Qt::CaseSensitivity) const
     int compare(const QByteArray &s, Qt::CaseSensitivity cs = Qt::CaseSensitive) const
         ^
make: *** [Metadata.o] Error 1
make: *** Waiting for unfinished jobs....
The command "make -j2" exited with 2.
```

**Comment by myst6re** [comment]
Related to #9 and #10

**Comment by myst6re** [comment]
Fixed in #10, not merged in master yet.
